subprojects/gradle/build-metrics/src/main/kotlin/com/avito/android/plugin/build_metrics/internal/cache/BuildCacheOperationListener.kt

```diff
@@ -147,6 +152,7 @@ internal class BuildCacheOperationListener(
             message.substringAfter(" response status ").substringBefore(':').trim()
                 .toInt()
         } else {
+            logger.warn("Unknown cache store error", error)
```

Comment:
Will we see those warnings at Grafana? Or when and how I need to react on them?

Reply:
These events are already in grafana as "unknown" errors.
I want to collect them to Sentry for further troubleshooting.